src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsBaseSpec.groovy

```diff
@@ -105,24 +106,28 @@ abstract class PriceFloorsBaseSpec extends BaseSpec {
         BidRequest.defaultBidRequest.tap { imp[0].video = Video.defaultVideo }
     }
 
-    protected void cacheFloorsProviderRules(PrebidServerService pbsService = floorsPbsService,
-                                            BidRequest bidRequest,
-                                            BigDecimal expectedFloorValue) {
-        PBSUtils.waitUntil({ pbsService.sendAuctionRequest(bidRequest).ext.debug.resolvedRequest.imp[0].bidFloor == expectedFloorValue },
+    protected void cacheFloorsProviderRules(BidRequest bidRequest,
```

Comment:
Please align parameter ordering in all 3 methods.
Parameter order in those 3 methods are all over the place now. Was there a reason to do it that way? All of them have 3 identical parameters, but their position is different between methods. This looks both weird and bad.